app/assets/javascripts/views/index_table_view.js.coffee

```diff
@@ -60,96 +48,10 @@ class ELMO.Views.IndexTableView extends ELMO.Views.ApplicationView
   unhighlight_partner_row: (event) ->
     $(event.target).closest('tbody').find('tr.hovered').removeClass('hovered')
 
-  # selects/deselects all boxes
-  select_all_clicked: (event) ->
-    event.preventDefault() if event
-
-    cbs = this.get_batch_checkboxes()
-
-    # Toggle the value of the select_all field
-    value = if @select_all_field.val() then '' else '1'
-    @select_all_field.val(value)
-
-    # check/uncheck boxes
-    cb.checked = value for cb in cbs
-
-    # update select all view elements
-    this.update_select_all_elements()
-
-    return false
-
-  # tests if all boxes are checked
-  all_checked: (cbs = this.get_batch_checkboxes()) ->
-    _.all(cbs, (cb) -> cb.checked)
-
-  # reset the alert element
-  reset_alert: ->
-    @alert.stop().hide().
-      removeClass('alert-danger alert-info alert-warning alert-success').removeAttr('opacity')
-
-  # updates the select all link to reflect the select_all field
-  update_select_all_elements: ->
-    label = if @select_all_field.val() then "deselect_all" else "select_all"
-    $('#select_all_link').html(I18n.t("layout.#{label}"))
-
-    this.reset_alert()
-
-    if @pages > 1 and @select_all_field.val()
-      msg = if @is_search then 'searched_rows_selected' else 'all_rows_selected'
-      @alert.html(I18n.t("index_table.messages.#{msg}", {count: @entries}))
-      @alert.addClass('alert-info').show()
-
-  # gets all checkboxes in batch_form
-  get_batch_checkboxes: ->
-    @form.find('input[type=checkbox].batch_op')
-
-  get_selected_count: ->
-    if @select_all_field.val()
-      @entries
-    else
-      _.size(_.filter(this.get_batch_checkboxes(), (cb) -> cb.checked))
-
-  get_selected_items: ->
-    @form.find('input.batch_op:checked')
-
   # event handler for when a checkbox is clicked
   checkbox_changed: (event) ->
```

Comment:
This method is no longer in the events hash so I'm not sure why it's here. Also are you sure this class needs to depend on the newer view? It's best to reduce coupling wherever possible.